README.md

```diff
@@ -57,11 +57,23 @@ INSTALLED_APPS = [
 ]
 ```
 
-You'll also need to set up two variables with the Google Cloud credentials:
-`RELE_GC_CREDENTIALS` and `RELE_GC_PROJECT_ID`.
+You'll also need to set up your settings.
 
-You can use `RELE_SUB_PREFIX` to add a prefix to all your subscription names, for
-example, with your project name.
+```python
+from google.oauth2 import service_account
+RELE = {
+    'GC_CREDENTIALS': service_account.Credentials.from_service_account_file(
```

Comment:
What do you think about to change this to `'GC_CREDENTIALS': ''` 
or `'GC_CREDENTIALS': os.environ.get('GC_CREDENTIALS')`